### ISSUE TITLE
Accessibility improvement for <textarea> form

### ISSUE THREAD

**Issue by sara-akhtar** [scniro/react-codemirror2#304]
There's an issue that's impacting multiple components in our application. The `<textarea>` form has no associated label which doesn't allow the browser to announce the control to a user that may need it. 

Recommendations on fixes: 
- If the control is a [labelable](https://html.spec.whatwg.org/multipage/forms.html#category-label) element, add a for attribute to a visible label with the id of this control (e.g. <label for="controlID">);
- Or, if the control is a labelable element, put the control inside a visible <label> element;
- Or, add an aria-labelledby attribute to the control. It must point to visible text on the page that is meaningful as a label;
- Or, if the design cannot have a visible label, provide a label using the aria-label attribute (e.g. aria-label="Activities");
- Or, if the design cannot have a visible label, use the title attribute to provide a label.

**Comment by sara-akhtar** [comment]
@scniro any updates on this bug?

**Comment by alestrunda** [comment]
By default, there is no label, but you can set it in the `options` as `screenReaderLabel` - will set it as `aria-label`.

**Comment by jinhua-ibm** [comment]
Hey @alestrunda:

Following up with your comment, and don't think `aria-label` is shown in <textarea>, can I have further instructions if possible?

https://stackblitz.com/edit/react-mkuohsnx?file=index.js


```react
<CodeMirror
  value="<h1>I ♥ react-codemirror2</h1>"
  options={{
    mode: 'xml',
    screenReaderLabel: 'jinhua',
    lineNumbers: true,
  }}
  onChange={(editor, data, value) => {}}
/>
```

**Comment by alestrunda** [comment]
@jinhua-ibm in that example you're using an old version of the lib - from `package.json` `"react-codemirror2": "^6.0.0"`. I was using `7.3.0` and it was working fine there. I'm not sure about other versions.
